companies-manifest.js

```diff
@@ -946,6 +951,91 @@ const companies = [
       },
     },
   },
+  {
+    key: 'https://www.unrealengine.com/',
+    similarWebGlobalRank: 5206,
+    name: 'Unreal Engine',
+    logo: UnrealEngineLogo,
+    style: {},
+    projects: {
+      unrealengine: {
+        title: 'Unreal Engine',
+        link: 'https://www.unrealengine.com/',
+        src: '/screenshots/thumbnails/www.unrealengine.com.jpg',
+        width: 1280,
+        height: 720,
+        internalUrl: 'unrealengine',
+      },
+    },
+  },
+  {
+    key: 'https://smallpdf.com',
+    similarWebGlobalRank: 783,
+    name: 'Smallpdf',
+    logo: SmallPdfLogo,
+    style: {},
+    projects: {
+      smallpdf: {
+        title: 'Smallpdf',
+        link: 'https://smallpdf.com',
+        src: '/screenshots/thumbnails/smallpdf.com.jpg',
+        width: 1280,
+        height: 720,
+        internalUrl: 'smallpdf',
+      },
+    },
+  },
+  {
+    key: 'https://overwatchleague.com',
+    similarWebGlobalRank: 22316,
+    name: 'Blizzard',
+    logo: OverWatchLeagueLogo,
+    style: {},
+    projects: {
+      overwatchleague: {
+        title: 'Overwatch League',
+        link: 'https://overwatchleague.com',
+        src: '/screenshots/thumbnails/overwatchleague.com.jpg',
+        width: 1280,
+        height: 720,
+        internalUrl: 'overwatchleague',
+      },
+    },
+  },
+  {
+    key: 'https://www.lauyan.com',
+    similarWebGlobalRank: 'N/A',
```

Comment:
Can you replace this with `Number.MAX_SAFE_INTEGER` in order to keep type consistency and to make it a little more obvious how sorting would work?